### PR TITLE
fr translation

### PR DESCRIPTION
**Pull request by waam**
Add french translation.

### CODE REVIEW

File: src/locale/lang/fr.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
+export default {
+  el: {
+    datepicker: {
+      now: 'Maintenant',
+      today: "Aujourd'hui",
```
Comment (Leopoldthecoder) [inline src/locale/lang/fr.js]: First, thank you for contributing! Here's a thing: some of the texts here are nested in a limited space. Take "Today" for example:
image
I don't think there is enough space for "Aujourd'hui".
Does the French language have any abbreviation or shorthand for "Aujourd'hui" and other long texts you provided here?

Comment (waam) [inline src/locale/lang/fr.js]: "Auj." (with the dot at the end) could be used. It's popular.
"Mnt" is one possible abbreviation for "Maintenant", but it sounds weird aside the "OK" button.

"Sélectionner" could be replaced by "Choisir".

Comment (Leopoldthecoder) [inline src/locale/lang/fr.js]: I see. So could you please replace these words with their appropriate abbreviations? Thanks.

File: src/locale/lang/fr.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+      cancel: 'Annuler',
+      error: 'Erreur'
+    },
+    upload: {
+      delete: 'Supprimer',
+      preview: 'Aperçu',
+      continue: 'Continuer'
+    },
+    table: {
+      emptyText: 'Aucune donné',
```
Comment (jaoued) [inline src/locale/lang/fr.js]: Aucune donnée

**Pull request merged**